docs/monoDrive_home/getting_started/Linux.md

```diff
@@ -30,11 +30,26 @@
       $ make
     </pre>
 
-1. Afterwards to launch the Scenario Editor or Simulator:
-  <pre>
+1. Afterwards to launch the Simulator:
+
+    ```
     ${UE4_ROOT}/Engine/Binaries/Linux/UE4Editor ${SIMULATOR_ROOT}/Simulator/VehicleAI.uproject
-  </pre>
-    Note that the editor may take a long time to compiling shaders, and may appear to get stuck at 95% or 99%. This is expected and you will need to wait for the compiling to finish. If no error is reported then the system is still compiling shaders, this only happens the first time opening. 
+    ```
+
+    To launch the Scenario Editor:
+
+    ```
+    ${UE4_ROOT}/Engine/Binaries/Linux/UE4Editor ${SIMULATOR_ROOT}/VehicleAI.uproject
```

Comment:
Step 6 here doesn't really make sense. The simulator and scenario editor are essentially the same command. Also I'm not sure that the final command even works?

Suggest just make this a single command to use the `UE4Editor` to open the `VehicleAI.uproject`.